ext/date/date_core.c

```diff
@@ -7192,13 +7212,15 @@ strftimev(const char *fmt, VALUE self,
 
 /*
  * call-seq:
- *    d.asctime  ->  string
- *    d.ctime    ->  string
+ *   asctime -> string
  *
- * Returns a string in asctime(3) format (but without "\n\0" at the
```

Comment:
I think we should keep the fact that this is equivalent to the system `asctime` format.

Reply:
Done (with link).